diff -Naur gimp-perl.orig/extra.c gimp-perl/extra.c
--- gimp-perl.orig/extra.c	2012-12-19 08:50:16.292278083 -0500
+++ gimp-perl/extra.c	2012-12-19 08:50:41.604282832 -0500
@@ -11,7 +11,7 @@
 
 /* need to factor these out, otherwise we always need gtk :( */
 #include <libgimp/gimp.h>
-#include <libgimpwidgets/gimpchainbutton.h>
+#include <libgimpwidgets/gimpwidgets.h>
 #include <libgimpbase/gimpbasetypes.h>
 
 #include "extra.h"
