diff -Naur gimp-perl.orig/config.pl gimp-perl/config.pl
--- gimp-perl.orig/config.pl	2012-12-19 08:50:16.292278083 -0500
+++ gimp-perl/config.pl	2012-12-19 09:33:19.614612115 -0500
@@ -10,6 +10,7 @@
 use ExtUtils::PkgConfig;
 %gimpcfg = ExtUtils::PkgConfig->find("gimp-2.0");
 %glibcfg = ExtUtils::PkgConfig->find("glib-2.0");
+%gtkcfg = ExtUtils::PkgConfig->find("gtk+-2.0");
 
 $^W=0;
 
@@ -29,8 +30,8 @@
 %cfg = (
    GIMP			=> $gimppath . $gimpbinname, 
    GIMPTOOL		=> $gimppath . "gimptool-2.0",
-   _GIMP_INC		=> $gimpcfg{"cflags"},
-   _GIMP_INC_NOUI	=> $gimpcfg{"cflags"},
+   _GIMP_INC		=> $gimpcfg{"cflags"}." ".$gtkcfg{"cflags"},
+   _GIMP_INC_NOUI	=> $gimpcfg{"cflags"}." ".$gtkcfg{"cflags"},
    _GIMP_LIBS		=> $pluginlibs,
    _GIMP_LIBS_NOUI	=> $gimpcfg{"libs"},
 
